docs/acceptance.md

```diff
@@ -1,68 +1,68 @@
 
 ### In General
-- [x] [#38](https://github.com/mozilla/Themer/issues/38) 
+- [x] [#38](https://github.com/mozilla/FirefoxColor/issues/38) 
   It should generally match Photon styles.
-- [ ] [#31](https://github.com/mozilla/Themer/issues/31) 
+- [ ] [#31](https://github.com/mozilla/FirefoxColor/issues/31) 
   It should specify that it is a Firefox Test Pilot experiment. (cf Send)
-- [ ] [#39](https://github.com/mozilla/Themer/issues/39) 
+- [ ] [#39](https://github.com/mozilla/FirefoxColor/issues/39) 
   It should have a list of mandatory links at the bottom of the page including any required privacy documentation.(cf Send)
-- [x] [#23](https://github.com/mozilla/Themer/issues/23) 
+- [x] [#23](https://github.com/mozilla/FirefoxColor/issues/23) 
   It should not be called ThemesRFun (or any derivation thereof).
-- [ ] [#15](https://github.com/mozilla/Themer/issues/15) 
+- [ ] [#15](https://github.com/mozilla/FirefoxColor/issues/15) 
   It should match final visual specs.
 
 ### Add-on UI
 - [x] It should be accessible through a browserAction.
-- [x] [#18](https://github.com/mozilla/Themer/issues/18) 
+- [x] [#18](https://github.com/mozilla/FirefoxColor/issues/18) 
   The browserAction should be a single color SVG icon that accepts a value of `fill=context-fill`
-- [x] [#18](https://github.com/mozilla/Themer/issues/18) 
+- [x] [#18](https://github.com/mozilla/FirefoxColor/issues/18) 
   The browserAction should not be a clone of the current [Photon customize icon](https://design.firefox.com/icons/viewer/#customize).
 - [x] Hitting the browserAction should direct the user to the TRF site
 
 ### For all desktop browser users
-- [ ] [#41](https://github.com/mozilla/Themer/issues/41) 
+- [ ] [#41](https://github.com/mozilla/FirefoxColor/issues/41) 
   It should be functionally responsive down to 640px.
 - [x] It should let users select colors for each individual component of the theming API via WYSIWYG editor.
 - [x] It should let users select background images or patterns from a preset list.
 - [x] It should let users select from preset themes using some kind of visual picker.
 - [x] It should let users see an in-content representation of how their theme would look in the Firefox Browser.
-- [x] [#42](https://github.com/mozilla/Themer/issues/42) 
+- [x] [#42](https://github.com/mozilla/FirefoxColor/issues/42) 
   It should let users undo or redo the changes they've made in an editing session.
 - [x] It should let users see a custom URL of the theme they've generated.
 
 ### For FF users with the TRF add-on
```

Comment:
Really stupid nit, but should we change "TRF" to "Firefox Color" or not even worth the effort?